resources/queries/targetedms/InstrumentSummaryByFolder.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
+       COUNT(ReplicateId.RunId) AS SkylineDocumentCount,
+       COUNT(DISTINCT ReplicateId) AS ReplicateCount,
+       MIN(AcquiredTime) AS FirstAcquisition,
+       MAX(AcquiredTime) AS LastAcquisition,
+       ReplicateId.RunId.Container,
+       InstrumentSerialNumber
+FROM targetedms.SampleFile
+WHERE
+      InstrumentSerialNumber IS NOT NULL
```

Comment:
I'd like to still show files without a serial number in the summary even if we can't offer QC folder links.

Reply:
OK